universal-login-contracts/test/fixtures/basicIdentity.js

```diff
@@ -0,0 +1,68 @@
+import ERC1077ApprovalScheme from '../../build/ERC1077';
+import MockToken from '../../build/MockToken';
+import MockContract from '../../build/MockContract';
+import ethers, {utils} from 'ethers';
+import {deployContract} from 'ethereum-waffle';
+import {addressToBytes32} from '../utils';
+import {OPERATION_CALL} from '../../lib/consts';
+import DEFAULT_PAYMENT_OPTIONS from '../../lib/defaultPaymentOptions';
+
+const {parseEther} = utils;
+const {gasPrice, gasLimit} = DEFAULT_PAYMENT_OPTIONS;
+
+export default async function basicIdentity(provider, wallet) {
+  const publicKey = addressToBytes32(wallet.address);
```

Comment:
I think we should separate wallet for a public, key otherwise it might become pretty tricky what is what.